Mastercard.Developer.OAuth1Signer.RestSharpV2/Mastercard.Developer.OAuth1Signer.RestSharpV2.csproj

```diff
@@ -0,0 +1,38 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <Version>1.0.0</Version>
```

Comment:
For simplicity, and because we release and publish the packages simultaneously I would keep one single and same version for the 3 projects: `1.3.0`

https://github.com/Mastercard/oauth1-signer-csharp/releases